roles/slurm/templates/notification_scripts/nodes.bash

```diff
@@ -1,23 +1,146 @@
 #!/bin/bash
 
```

Comment:
just a suggestion:
since I see that you have a lot of logger commands commented for debugging and since this script is currently not using arguments, I would suggest adding at the top something like
```bash
_debug=false
[[ -n ${1} ]] && [[ "${1}" == "-d" ]] && _debug=true
```

so then instead of
```bash
#logger --id="${$}" -s "Got new node state stored in ${node_state_file_new}."
```
you could use a line that triggers debugging when called with `-d`
```bash
${_debug} && logger --id="${$}" -s "Got new node state stored in ${node_state_file_new}."
```

Reply:
Point taken; update to this PR will follow...